junit-platform-engine/src/main/java/org/junit/platform/engine/support/hierarchical/SingleTestExecutor.java

```diff
@@ -1,78 +0,0 @@
-/*
- * Copyright 2015-2018 the original author or authors.
- *
- * All rights reserved. This program and the accompanying materials are
- * made available under the terms of the Eclipse Public License v2.0 which
- * accompanies this distribution and is available at
- *
- * http://www.eclipse.org/legal/epl-v20.html
- */
-
-package org.junit.platform.engine.support.hierarchical;
-
-import static org.apiguardian.api.API.Status.MAINTAINED;
-import static org.junit.platform.commons.util.BlacklistedExceptions.rethrowIfBlacklisted;
-import static org.junit.platform.engine.TestExecutionResult.aborted;
-import static org.junit.platform.engine.TestExecutionResult.failed;
-import static org.junit.platform.engine.TestExecutionResult.successful;
-
-import org.apiguardian.api.API;
-import org.junit.platform.engine.TestExecutionResult;
-import org.opentest4j.TestAbortedException;
-
-/**
- * {@code SingleTestExecutor} encapsulates the execution of a single test
- * wrapped in an {@link Executable}.
- *
- * @since 1.0
- * @see #executeSafely(Executable)
- */
-@API(status = MAINTAINED, since = "1.0")
```

Comment:
Just realized this has status _maintained_... In that case, I'll resurrect and deprecate it.